.gitignore

```diff
@@ -191,3 +191,4 @@ sdk/template/template-dpg/src/src
 # sshkey
 sdk/**/sshKey
 sdk/**/sshKey.pub
+/sdk/translation/ai-translation-document-rest/nbproject/private/
```

Comment:
Could you put that in the .gitignore for your particular folder instead?

Reply:
Deleted

Reply:
I still see it here, and nbproject committed - could we clean that up? Many thanks!

Reply:
Sorry, looks like it got created when I regenerated the SDK. removed that line from .gitignore and deleted the folder nbproject